config/filter.d/asterisk.conf

```diff
@@ -25,14 +25,24 @@ failregex = ^(%(__prefix_line)s|\[\]\s*)%(log_prefix)s Registration from '[^']*'
             ^(%(__prefix_line)s|\[\]\s*)%(log_prefix)s Host <HOST> failed MD5 authentication for '[^']*' \([^)]+\)$
             ^(%(__prefix_line)s|\[\]\s*)%(log_prefix)s Failed to authenticate (user|device) [^@]+@<HOST>\S*$
             ^(%(__prefix_line)s|\[\]\s*)%(log_prefix)s hacking attempt detected '<HOST>'$
-            ^(%(__prefix_line)s|\[\]\s*)%(log_prefix)s SecurityEvent="(FailedACL|InvalidAccountID|ChallengeResponseFailed|InvalidPassword)",EventTV="([\d-]+|%(iso8601)s)",Severity="[\w]+",Service="[\w]+",EventVersion="\d+",AccountID="(\d*|<unknown>)",SessionID=".+",LocalAddress="IPV[46]/(UDP|TCP|WS)/[\da-fA-F:.]+/\d+",RemoteAddress="IPV[46]/(UDP|TCP|WS)/<HOST>/\d+"(,Challenge="[\w/]+")?(,ReceivedChallenge="\w+")?(,Response="\w+",ExpectedResponse="\w*")?(,ReceivedHash="[\da-f]+")?(,ACLName="\w+")?$
+            ^(%(__prefix_line)s|\[\]\s*)%(log_prefix)s <HOST> tried to authenticate with nonexistent user.+$
+            ^(%(__prefix_line)s|\[\]\s*)%(log_prefix)s <HOST> failed to authenticate as.+$
```

Comment:
Do you have a most obscure but real example?

On May 21, 2016 8:22:15 PM EDT, Rob Thomas <notifications@github.com> wrote:
>> @@ -25,14 +25,24 @@ failregex =
>^(%(__prefix_line)s|\[\]\s*)%(log_prefix)s Registration from '[^']*'
>>              ^(%(__prefix_line)s|\[\]\s*)%(log_prefix)s Host <HOST>
>failed MD5 authentication for '[^']*' \([^)]+\)$
>>              ^(%(__prefix_line)s|\[\]\s*)%(log_prefix)s Failed to
>authenticate (user|device) [^@]+@<HOST>\S*$
>>              ^(%(__prefix_line)s|\[\]\s*)%(log_prefix)s hacking
>attempt detected '<HOST>'$
>> -            ^(%(__prefix_line)s|\[\]\s*)%(log_prefix)s
>SecurityEvent="(FailedACL|InvalidAccountID|ChallengeResponseFailed|InvalidPassword)",EventTV="([\d-]+|%(iso8601)s)",Severity="[\w]+",Service="[\w]+",EventVersion="\d+",AccountID="(\d*|<unknown>)",SessionID=".+",LocalAddress="IPV[46]/(UDP|TCP|WS)/[\da-fA-F:.]+/\d+",RemoteAddress="IPV[46]/(UDP|TCP|WS)/<HOST>/\d+"(,Challenge="[\w/]+")?(,ReceivedChallenge="\w+")?(,Response="\w+",ExpectedResponse="\w*")?(,ReceivedHash="[\da-f]+")?(,ACLName="\w+")?$
>> +            ^(%(__prefix_line)s|\[\]\s*)%(log_prefix)s <HOST> tried
>to authenticate with nonexistent user.+$
>> +            ^(%(__prefix_line)s|\[\]\s*)%(log_prefix)s <HOST> failed
>to authenticate as.+$
>
>No, as that can be, literally, anything. Unicode, binary, anything.  
>
>---
>You are receiving this because you commented.
>Reply to this email directly or view it on GitHub:
>https://github.com/fail2ban/fail2ban/pull/1409/files/644fe684cc29f8b851989c3cbc76d7c5d880c703#r64142203